tests/e2e/chaos/chainlink-relay-sol.yaml

```diff
@@ -9,14 +9,15 @@ charts:
   chainlink:
     index: 4
     values:
-      replicas: 19
+      replicas: 5
       chainlink:
         image:
-          image: "public.ecr.aws/chainlink/chainlink"
-          version: "develop.4bc99afb22dca23469d9a7bc2bdfa7ea49028ae6"
+          image: "public.ecr.aws/z0b1w9r9/chainlink"
+          version: "develop"
       env:
         eth_url: "ws://sol:8990"
         eth_disabled: "true"
+        SOLANA_ENABLED: "true"
         USE_LEGACY_ETH_ENV_VARS: "true"
         feature_external_initiators: "true"
         FEATURE_OFFCHAIN_REPORTING2: "true"
```

Comment:
Might be good to double check some of these env vars? the pulumi env has the latest ones I've been using: https://github.com/smartcontractkit/chainlink-solana/blob/develop/ops/Pulumi.localnet.yaml#L19

`ETH_DISABLED` and `USE_LEGACY_ETH_ENV_VARS` were deprecated in favor of:
    - EVM_ENABLED = false
    - EVM_RPC_ENABLED = false
    
also, i don't think `external_initiators` need to be enabled still? (unless that is used for something i'm not aware about)